tests/containers/podman_netavark.pm

```diff
@@ -24,7 +24,7 @@ sub is_cni_in_tw {
 # podman >=4.8.0 defaults to netavark
 sub is_cni_default {
     my $podman_version = get_podman_version();
-    return package_version_cmp($podman_version, '4.8.0') < 0 || is_sle_micro('<6.0');
+    return package_version_cmp($podman_version, '4.8.0') < 0 || is_sle_micro('<5.5');
```

Comment:
IMHO in this check is missing the same `...NetworkBackend...=~ "cni"`, that is present in `is_cni_in_tw` test.
In this actual status the check doesn't consider the current default.
I wish to update it accordingly